domains/tommi.json

```diff
@@ -0,0 +1,14 @@
+{
+  "owner": {
+    "username": "tommi",
+    "email": "surfing@tommi.space",
+    "GitHub": "xplosionmind",
```

Comment:
username needs to be your github username, remove github line and replace "tommi" with your github username (also sorry for late reply)

Reply:
Is there a way to use tommi.is-a.dev?